Jenkinsfile

```diff
@@ -79,9 +85,9 @@ pipeline {
     stage('check AutoPas integration') {
       agent { label 'atsccs11' }
       stages {
-        stage('build with autopas') {
-          parallel {
-            stage('build sequential') {
+        //stage('build with autopas') {
```

Comment:
```suggestion
        // parallel build currently disabled, because of OOM errors on atsccs11, when compiling two builds in parallel.
        //stage('build with autopas') {
```